Makefile

```diff
@@ -14,3 +14,52 @@ compose_up:
 
 compose_down:
 	docker-compose -f docker/docker-compose.yml down
+
+# Development Setup
+.venv:
+	python3 -m venv .venv
+	. .venv/bin/activate
+	python3 -m pip install -r requirements-dev.txt
+
+.git/hooks/pre-commit: .venv
+	python3 -m pre_commit install
+
+.env:
```

Comment:
Can we combine this into the venv one and call it `setup_local` or something? It's a bit too granular right now.

Reply:
I wouldn't mind something like:

```
_PHONY: setup_local
setup_local: .venv .env .git/hooks/pre-commit
```

And remove the current `precommit` target, then make everything else (that needs to) depend on `setup_local`. I do like to keep targets for specific files when there are actual files to be built.

Thoughts?

Reply:
I've made these changes, as well as switching the `development.env` template file to use the docker-hosted postgres by default.